### ISSUE TITLE
Enable the support of Authentication filter in mpjwt-1.x features

### ISSUE THREAD

**Issue by chunlongliang-ibm** [OpenLiberty/open-liberty#4714]
This feature enables the support of authFilter inside mpJwt-1.x features, so if mpJwt-1.x feature is enabled, mpJwt-1.x feature has option to decide if a request should be authenticated with JWT, or can be fallbacked to other available authentication methods. without this function, if mpJwt-1.x is enabled, all requests must be authenticated with JWT per MP-JWT specification.

This epic resolves https://bigblue.aha.io/features/SEC-11

**Comment by chunlongliang-ibm** [comment]
@gkwan-ibm 
WAD-- Not required. This is to update meta-type to define a reference to authnFilter, and update runtime to honor the configuration attribute.
Test and Demo -- Both L2 and L3 knows how authFilter working.
SVT -- Not Required
Queues -- Retain PMR and APAR: handled by WL3WSS.Salesforce Q name:  WAS L3: Security SSO

**Comment by chunlongliang-ibm** [comment]
@gscottj , there is no Accessibility issue

**Comment by dave-waddling** [comment]
**FAT Focal Approval**: Please can you complete the Feature Test Summary issue (#4721) that has been added to your Epic. This issue is what is used to review the FAT testing and obtain the FAT focal approval. Many thanks.

**Comment by bsbyrd1** [comment]
My approval is based on the sizing of the ID work, and the expectation that the ID work can be completed before the last ID translation drop.

**Comment by dave-waddling** [comment]
**FAT Focal Approval:** Hi, just a reminder that the Feature Test Summary issue (#4721) requires completion before approval can be granted.

**Comment by rumanaHaque** [comment]
No SVT required for this Feature, so marking the svt approval.

**Comment by chunlongliang-ibm** [comment]
[STE_for_MP_JWT.pptx](https://github.com/OpenLiberty/open-liberty/files/2311351/STE_for_MP_JWT.pptx)

**Comment by marikaj123** [comment]
Added the STE approval on behalf of Swati.

**Comment by dave-waddling** [comment]
**FAT Focal Approval:** The Feature Test Summary looks good and the last pair test cases were able to be tested sufficiently without requiring a full SOE (they were platform agnostic and incredibly basic). Approval granted.

**Comment by ayoho** [comment]
@chunlongliang-ibm and @arunavemulapalli: Sending another reminder - can this be closed?

**Comment by chunlongliang-ibm** [comment]
This feature has been completed.
